run_test.sh

```diff
@@ -55,11 +71,12 @@ run_with_expected_output() {
         sed '/^Sign command:/d' -i test_cmd_output.log
         sed '/^Signing key:/d' -i test_cmd_output.log
         sed '/^Public certificate (MOK):/d' -i test_cmd_output.log
-        sed '/^Signing module \/var\/lib\/dkms\/dkms_test\/1.0\/build\/dkms_test.ko$/d' -i test_cmd_output.log
         sed '/^Certificate or key are missing, generating them using update-secureboot-policy...$/d' -i test_cmd_output.log
         sed '/^Certificate or key are missing, generating self signed certificate for MOK...$/d' -i test_cmd_output.log
+        sed '/^kmodsign: CMS_add1_signer: Success$/d' -i test_cmd_output.log
         if [[ "${NO_SIGNING_TOOL}" = "1" ]]; then
             sed "/^Binary .* not found, modules won't be signed$/d" -i test_cmd_output.log
+            #sed "/^Signing module \/var\/lib\/dkms\/dkms_test\/1.0\/build\/dkms_test.ko$/d" -i test_cmd_output.log
```

Comment:
Why is this commented out?

Reply:
If there is no signing tool, there should not be "Signing module ..." line in the test_cmd_output.log.
But this line will appear when the --no-signing-tool argument is added to ./run_test.sh to simulate the absence of the sign tool in an environment where the sign tool does exist.
I commented out this line just to make it easier to debug in such environment.

Reply:
I see - it might be worth adding a small comment just above it then `# Uncomment the following like to $ACHIEVE_TASK`